### DISCUSSION TITLE
Device Streams Functionality

### DISCUSSION THREAD

**Discussion by mwspracklen** [Azure/azure-iot-sdk-c#2648] [Q&A]
Hello, the [MS learn iot hub device streams documentation](https://learn.microsoft.com/en-us/azure/iot-hub/iot-hub-device-streams-overview), says that the C device client sdk supports device streams. However, I cannot find anything in this repo indicating that is actually the case. Is the MS learn documentation incorrect?

**Comment by jpwgc** [comment]
I had the same question but per this comment https://github.com/Azure/azure-iot-sdk-c/issues/2378#issuecomment-1261153389 there are no plans to bring Device Streams GA, so I wouldn't be surprised if the documentation is wrong.
